content/en/containers/docker/apm.md

```diff
@@ -330,3 +393,8 @@ Refer to the [language-specific APM instrumentation docs][3] for tracer settings
 [1]: https://app.datadoghq.com/organization-settings/api-keys
 [2]: /tracing/guide/security/#replace-rules
```

Comment:
```suggestion
[2]: /tracing/configure_data_security/#replace-tags
```

Reply:
Looks like this link isn't valid anymore, and this is the page it redirects to. This is my best guess at what section the link should open.